examples/resources/illumio-cloudsecure_k8s_cluster_onboarding_credential/resource.tf

```diff
@@ -0,0 +1,18 @@
+resource "illumio-cloudsecure_k8s_cluster_onboarding_credential" "example" {
+  illumio_region = "aws-us-west-2"
+  name           = "Dev clusters AWS US"
+
+  # Optional attributes
+  description = "Credential to onboard EKS dev clusters in AWS US regions"
+}
+
+output "example_client_id" {
+  value       = illumio-cloudsecure_k8s_cluster_onboarding_credential.example.client_id
+  description = "The client_id to use to onboard k8s clusters."
+}
+
+output "example_client_secret" {
+  value       = illumio-cloudsecure_k8s_cluster_onboarding_credential.example.client_secret
+  description = "The client_secret to use to onboard k8s clusters."
+  sensitive   = true
+}
```

Comment:
We could, but it's not necessary. And I want to keep the example simple.
I added variables to show how to get the client_id/client_secret, because generating those is the only purpose of this resource.
The doc does document `created_at` so IMHO it's sufficient.